Jenkinsfile

```
@@ -959,7 +959,8 @@ pipeline {
                                                   'libpmemobj protobuf-c ' +
                                                   'spdk-devel libfabric-devel '+
                                                   'pmix numactl-devel ' +
-                                                  'libipmctl-devel' +
+                                                  'libipmctl-devel ' +
+						  'python36-tabulate' +
```

If you need to push again for any other reason:
```suggestion
						    'python36-tabulate' +
```